resources/locale.json

```diff
@@ -272,7 +272,17 @@
         "UI_PEACOCK_GAMECHANGERS_GLOBAL_HEADKILLONLY_NAME": "Head Attack Only",
         "UI_PEACOCK_GAMECHANGERS_GLOBAL_HEADKILLONLY_DESC": "All targets must be eliminated by using headshots or throwing lethal melee weapons.",
         "UI_MENU_PAGE_STORY_CUTSCENE_SEASON2_STORY_BLOCK_09_TITLE": "All According to Plan",
-        "UI_MENU_PAGE_STORY_CUTSCENE_SEASON2_STORY_BLOCK_10_TITLE": "A Trip to Paradise"
+        "UI_MENU_PAGE_STORY_CUTSCENE_SEASON2_STORY_BLOCK_10_TITLE": "A Trip to Paradise",
+        "UI_PEACOCK_CASPER": "The Aberratious Stravaig",
+        "UI_PEACOCK_CASPER1": "The Aberratious Stravaig - Level 1",
+        "UI_PEACOCK_CASPER2": "The Aberratious Stravaig - Level 2",
+        "UI_PEACOCK_CASPER3": "The Aberratious Stravaig - Level 3",
+        "UI_PEACOCK_CASPER_OBJ_LVL3_CLUBOWNERKILL_DESC": "Eliminate Daniel Fleischer in Rolf Hirschmüller's disguise",
+        "UI_PEACOCK_GAMECHANGERS_EDGY_CASPER_CYCLE_NAME": "Reinacrnators",
```

Comment:
I'll play it in the next few days. But there's a typo here, could you also make a PR for the image pack?

Reply:
Darn, I knew I will get an F- in English